.changeset/lovely-buttons-shout.md

```diff
@@ -0,0 +1,5 @@
+---
+'sku': patch
+---
+
+Add metrics for core scripts
```

Comment:
Might be worth adding a note here to install the package, rather than letting them find out the hard way?

Reply:
Was just adding this haha. Is what I added enough or should we add more reasoning?

Reply:
I'd say go a step further and add the commands for both yarn and npm here, so it's literally just a copy and paste.